docs/source/changelog/0.5.0-changelog.rst

```diff
@@ -1,3 +1,9 @@
+******
+v0.5.0
+******
+
+:Date: April 1, 2021
+
```

Comment:
Since this is being updated, can you run your version of the script and use its output for the file as the new changelog? At least this way the latest documentation will be correct for the contributors of the release.